build_release_image.sh

```diff
@@ -0,0 +1,119 @@
+help() {
+    echo "Build an Openshift Release Image"
+    echo "Usage:                    build_release_image.sh [options] <Images> <quay.io username>"
+    echo ""
+    echo "Options:"
+    echo "-h, --help                show this message"
+    echo "-t, --tag <tag>           push to a custom tag in your origin release image repo, default: mco"
+    echo ""
+    echo "Images: you must include at least one"
+    echo "-m, --mco                 build and push dependancies for mco to quay.io/<username>/image then"
+    echo "                              build a release image from that repo."
+    echo "-c, --capo [tag]          build a capo image locally, push it to quay.io/<username>/capo:[tag] then"
+    echo "                              build a release image from that repo."
+}
+
+TAG="mco"
```

Comment:
This should probably default to something else than `mco`. What do you think of `custom` ?